be/src/exec/pipeline/exchange/mcast_local_exchange.h

```diff
@@ -0,0 +1,132 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021 StarRocks Limited.
+
+#include "column/chunk.h"
+#include "exec/pipeline/source_operator.h"
+
+namespace starrocks {
+namespace pipeline {
+
+// ===== exchanger =====
+class MCastLocalExchanger {
```

Comment:
Oh, man, don't use short-name plz

Reply:
OK. But it really hurts eyes.

Reply:
You can use short-name in some unimportant place, but I think class name need a good name to avoid readers guess

Reply:
Yes sir!